air-quality-backend/src/etl/in_situ/openaq_adapter.py

```diff
@@ -73,11 +103,22 @@ def _sort(in_situ_data_for_city, city_name, input_lat, input_lon):
             # and sorted_cities_measurement[i]["date"]["utc"] has changed.
             chosen_place = measurement_location
             current_time = measurement_date
-            formatted_cities_measurement.append(create_document(measurement, city_name))
+            formatted_cities_measurement.append(
+                _create_document(measurement, city_name)
+            )
             to_create_document = False
 
-        if measurement["value"] != -1:
+        if (
+            measurement["value"] != -1
```

Comment:
If we're pulling out `measurement["value"]` and `measurement["parameter"]` more than once it might be nicer to name them as local variables to make it clear what they are like on line 85